### PR TITLE
simplenote: init at 1.0.6

### PR DESCRIPTION
**Pull request by dyrnade**
###### Motivation for this change
Simplenote Package 

###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x ] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/misc/simplenote/default.nix [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+, libgnome_keyring3, gdk_pixbuf, gvfs, cairo, cups, expat, libgpgerror, nspr
+, nss, xorg, libcap, systemd, libnotify ,libXScrnSaver, pkgs }:
+
+ stdenv.mkDerivation rec {
+
+  name = "simplenote-${pkgver}";
+  pkgver = "1.0.6";
+
+  src = fetchurl {
+    url = "https://github.com/Automattic/simplenote-electron/releases/download/v${pkgver}/${name}.deb";
```
Comment (rasendubi) [inline pkgs/applications/misc/simplenote/default.nix]: Can't we build it from source?

File: pkgs/applications/misc/simplenote/default.nix [hunk 2]
```diff
@@ -0,0 +1,55 @@
+{ fetchurl, stdenv, lib, zlib, glib, alsaLib, dbus, gtk2, atk, pango, freetype, fontconfig
+, libgnome_keyring3, gdk_pixbuf, gvfs, cairo, cups, expat, libgpgerror, nspr
+, nss, xorg, libcap, systemd, libnotify ,libXScrnSaver, pkgs }:
```
Comment (rasendubi) [inline pkgs/applications/misc/simplenote/default.nix]: `s/ ,/, /`

don't pass `pkgs` in, better use `gnome3`

**Comment by rasendubi** [comment]
Please also clean up your history.

**Comment by rasendubi** [comment]
@dyrnade why close?

**Comment by dyrnade** [comment]
My mistake, i will redo pr. Sorry

**Pull request closed without merging**